eloquentjs/20_2_directory_creation/src/modules/helpers.ts

```diff
@@ -0,0 +1,29 @@
+import {resolve, sep} from "node:path"
+import { IncomingMessage } from "node:http";
+import { WriteStream } from "node:fs";
+
+
+const baseDirectory = process.cwd();
+
+function pipeStream(from: IncomingMessage, to: WriteStream) {
+    return new Promise((resolve, reject) => {
+        from.on("error", reject);
+        to.on("error", reject);
+        to.on("finish", resolve);
+        from.pipe(to);
+    });
+}
+
+function urlPath(url: string) {
```

Comment:
this name makes it sound like this is the url path and not a function for getting the url path. `extractUrlPath` would be better